index.html

```diff
@@ -651,35 +651,128 @@ <h4>Registration Information</h4>
                             <tr>
                                 <th>Vocabulary term</th>
                                 <th>Description</th>
-                                <th>Assignment</th>
+                                <th>Client Assignment</th>
```

Comment:
not clear if client is "Registrant" or "Querier/Consumer".  I would add another column and distinguish these two.  A Registrant "sets" the ttl, for example, but a Consumer can only read it from the Directory.  Also, Server should probably be "Directory".